drivers/adc-dac/ad5592r/ad5593r.h

```diff
@@ -36,6 +36,17 @@
 #include "stdint.h"
 #include "ad5592r-base.h"
 
+#define AD5593R_MODE_CONF		(0 << 4)
+#define AD5593R_MODE_DAC_WRITE		(1 << 4)
+#define AD5593R_MODE_ADC_READBACK	(4 << 4)
+#define AD5593R_MODE_DAC_READBACK	(5 << 4)
+#define AD5593R_MODE_GPIO_READBACK	(6 << 4)
+#define AD5593R_MODE_REG_READBACK	(7 << 4)
+
+#define STOP_BIT	1
```

Comment:
tbh i would drop this STOP_BIT, it's pretty clear what it does, no need to have a macro for it

alternatively, prefix it, because at some point there is going to be a name clash